test/CMakeLists.txt

```diff
@@ -287,7 +287,10 @@ endif()
 if(LINUX
    AND UMF_BUILD_SHARED_LIBRARY
    AND UMF_POOL_SCALABLE_ENABLED)
-    add_umf_test(NAME init_teardown SRCS test_init_teardown.c)
+    add_umf_test(
+        NAME init_teardown
+        SRCS test_init_teardown.c
+        LIBS dl)
```

Comment:
When we had a similar issue in the past, @bratpiorka fixed it by adding `dl` dependency to utils static library. Why a similar fix does not work here?
If `init-teardown` test uses `utils`, should it be linked with `utils`?

Reply:
The `init-teardown` test does not use `utils`.